docs/help/faq.md

```diff
@@ -93,6 +93,21 @@ These issues are most commonly caused by programming errors, but can be related
 
 A bus error is almost always related to a process requesting more memory than is available. In other words, the job ran out of memory. The simplest solution is to increase the requested memory for the job. It may help to learn more about [SLURM](../cheaha/slurm/introduction.md) and [job efficiency](../cheaha/job_efficiency.md).
 
+#### I copied a script from a Windows machine to Cheaha and am getting a bad interpreter error
+
+Bad interpreter errors usually mean that there are symbols in the script that the shell interpreter cannot read correctly. This is usually due to the difference in newline symbols used in Windows (`\r\n`) and Unix (`\n`). Once these newline symbols are changed, the script should be able to run on Cheaha.
+
+To convert a script written in Windows to be compatible on Linux, use the `dos2unix` command. Just give it the name of the script to convert and it should convert it correctly.
```

Comment:
Can you show an example of the usage with an example script name? It's simple sure, but not always to folks who aren't aware the systems have different line endings.